instrumentation/sidekiq/lib/opentelemetry/instrumentation/sidekiq/middlewares/client/tracer_middleware.rb

```diff
@@ -29,6 +32,18 @@ def call(_worker_class, job, _queue, _redis_pool)
 
             private
 
+            def span_name(job)
+              if config[:enable_job_class_span_names]
+                job['wrapped']&.to_s || job['class']
```

Comment:
```suggestion
                "#{job['wrapped']&.to_s || job['class']} enqueue"
```
@robertlaurin and I looked at some traces with the job class as the span name, and it is a little confusing to see the same span name for the producer and consumer. It is much more readable to see `Foo enqueue` and `Foo process`, and "enqueue fits the domain language better [than send]" (quoting @robertlaurin).

Reply:
Adjusted, went with `enqueue` only for job class named spans until the conventions are updated for job systems.